CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ Changelog
 * **2015-04-07**: Class metadata now validates that you can not map the UUID on
                   Documents that are not referenceable. Either set your
                   documents `referenceable=true` or remove the UUID mapping.
+* **2015-03-04**: Translated properties now take into account the "property"
+                  mapping (allowing customiation of the PHPCR property name).
```

Comment:
Will fix and merge  next chance I have

On 6 July 2015 09:28:47 CEST, David Buchmann <notifications@github.com> wrote:
>> @@ -14,6 +14,8 @@ Changelog
>>  * **2015-04-07**: Class metadata now validates that you can not map
>the UUID on
>>                    Documents that are not referenceable. Either set
>your
>>                    documents `referenceable=true` or remove the UUID
>mapping.
>> +* **2015-03-04**: Translated properties now take into account the
>"property"
>> +                  mapping (allowing customiation of the PHPCR
>property name).
>
>s/customiation/customization
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/doctrine/phpcr-odm/pull/647/files#r33910874

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.